Octokit/Models/Response/PullRequest.cs

```diff
@@ -48,6 +49,11 @@ public PullRequest(Uri url, Uri htmlUrl, Uri diffUrl, Uri patchUrl, Uri issueUrl
         }
 
         /// <summary>
+        /// The universal Id (different from the pull request number) this pull request.
+        /// </summary>
+        public int Id { get; protected set; }
```

Comment:
In terms of running the integration tests, you need to configure your development environment (and is strongly recomended you setup a separate github account to use for the integration tests).  More info is documented [here in the contributors guide](https://github.com/octokit/octokit.net/blob/master/CONTRIBUTING.md#slow-tests)